librz/cons/pal.c

```diff
@@ -521,28 +521,32 @@ RZ_API void rz_cons_pal_show(void) {
 typedef struct {
 	int val;
 	const char *str;
-} RAttrStr;
+} RzAttrStr;
 
-RZ_API void rz_cons_pal_list(int rad, const char *arg) {
-	char *name, **color;
+RZ_API RzList *rz_cons_pal_list(int rad, const char *arg) {
```

Comment:
Add ownership mark please. Also swithc `rad` to the `RzOutputMode`

Reply:
> Also swithc rad to the RzOutputMode

Yeah, about that. 
There was this weird edge case inside that, a `case 'c'` which handled `ecc`. `ecc` is a command which prints the shows the palette in CSS. I just put that under `RZ_OUTPUT_MODE_LONG`. See if that sounds alright.

Reply:
I think it's better to make this a separate command. @ret2libc what do you think?